init.ts

```diff
@@ -106,7 +107,15 @@ setup(app, {
 })
 	.tap(createSuperuser)
 	.then(({ startServer }) => {
-		return startServer(process.env.PORT || 1337).return();
+		startServer(process.env.PORT || 1337).return();
```

Comment:
```suggestion
		return startServer(process.env.PORT || 1337).return();
```
(or drop the `{}` for implicit return)